charts/tidb-backup/templates/scripts/_start_backup.sh.tpl

```diff
@@ -19,3 +19,11 @@ uploader \
   --bucket={{ .Values.backup.gcp.bucket }} \
   --backup-dir=/data/${dirname}
 {{- end }}
+
+{{- if .Values.backup.ceph }}
+uploader \
+  --cloud=ceph \
+  --bucket={{ .Values.backup.ceph.bucket }} \
```

Comment:
Is this the S3 API? If so can we just configure this as AWS rather than ceph?

Reply:
Yes. We use rook ceph operator with the S3 API. The config is a bit different, like endpoint
https://github.com/tennix/tidb-cloud-backup/blob/master/pkg/setup.go#L111

Reply:
endpoint is actually supported by AWS though, its just that most users won't be specifying it but instead specifying a region. It looks like the main difference is that `--cloud=ceph` is translated to some different default configuration values:

```
WithS3ForcePathStyle(true).
		WithDisableSSL(true).
WithMaxRetries(20)
```

We should consider an upload that is of the format

```
aws
  bucket: backup
  ceph: true
```

Where `ceph: true` creates those above option defaults.

I am also okay with keeping it as is if you still think that is the best.

Reply:
OK. I'd like to keep it now. After finishing the backup feature of TEP, I may change it in the future.
Another difference is that I check and create the bucket on rook ceph object store, see https://github.com/tennix/tidb-cloud-backup/blob/master/pkg/setup.go#L119.
Ceph also support swift protocal, I'm not going to support it now.